docs/recipes/react.md

```diff
@@ -0,0 +1,69 @@
+# React testing
+
+With 0.10.0, AVA stopped including your `.babelrc` (for reasons outlined in [this PR](https://github.com/sindresorhus/ava/pull/398)), which means you can't use JSX in your test/spec files. Note that this doesn't apply to your source files, they can still be written in JSX!
```

Comment:
I think we are pretty close to agreement on how it should look. But the implementation is not written yet.